src/app/shared/log-in/container/log-in-container.component.ts

```diff
@@ -51,8 +53,12 @@
   /**
    * Find the correct component based on the AuthMethod's type
    */
-  getAuthMethodContent(): string {
-      return rendersAuthMethodType(this.authMethod.authMethodType);
+  getAuthMethodContent(): Component {
+    const component = rendersAuthMethodType(this.authMethod.authMethodType);
+    if (component === undefined) {
+      this.elRef.nativeElement.classList.add('d-none');
+    }
+    return component;
```

Comment:
If the problem is that the `component` is undefined, it would be better to filter out all the `authMethodType` that are not configured. To achieve this behavior, you should filter them inside the `log-in-component` and add the related method inside the `login-in.methods-decorator`.
Do you agree @alexandrevryghem?

Reply:
Thnx for the review @vins01-4science, I indeed do agree that it would be even better to filter them out earlier so that it never even generates a `LogInContainerComponent` for `AuthMethod`s without a `Component`. But I would prefer to just check if the `Component` is undefined though, so that it works dynamically, independently of which (future) `AuthMethod`s are added.